compiler/pipes/calc-bad-vars.cpp

```diff
@@ -591,7 +591,25 @@ class CalcBadVars {
         func->wait_prev = to_parents[func];
       }
     }
+
+    auto get_call_resumable_path = [&](FunctionPtr func) {
```

Comment:
Boring comment - What if func will be null here?

Reply:
It cannot be null there because it's only called with valid function, line 612. I've added assert.